src/HTTPConnection.cpp

```diff
@@ -206,7 +207,7 @@ int HTTPConnection::updateBuffer() {
         } else {
           // An error occured
           _connectionState = STATE_ERROR;
-          HTTPS_LOGE("An receive error occured, FID=%d", _socket);
+          HTTPS_LOGE("An receive error occured, FID=%d, SSL_error=%d", _socket, readReturnCode); 
```

Comment:
That's not necessarily the SSL error, only if this is an HTTPSConnection.

```suggestion
          HTTPS_LOGE("An receive error occurred, FID=%d, code=%d", _socket, readReturnCode); 
```

Reply:
Unfortunately I deleted the logs and it was quite some time ago. So can not really tell, if I saw.

Reply:
This morning I realized I can reproduce so can check how the log looks like. So here it is.
Looks like the SSL error appears.

```[HTTPS:D] << Websocket.send()
[HTTPS:D] >> Websocket.send(): length=143
[HTTPS:D] << Websocket.send()
[HTTPS:D] Data on Socket FID=61
[HTTPS:D] SSL_read error: 2
[HTTPS:E] An receive error occured, FID=61, SSL_error=-1
[HTTPS:I] Connection closed. Socket FID=61
[HTTPS:D] Free headers
[HTTPS:D] Free WS Handler